services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The parameters `genres_ids` and `actors_ids` are initialized with `None`. Consider using an empty list as a default value to avoid potential issues with `NoneType` operations. However, be cautious as using mutable default arguments can lead to unexpected behavior if not handled properly.